docs/InstallGuide.md

```diff
@@ -1,129 +1,136 @@
-# OpenVNet #
+# OpenVNet Installation Guide
 
-# Preliminary Operations and Installation #
+## Overview
 
-> **Note:**
-This document explains the installation of OpenVNet.  However, it was
-written for an older version of OpenVNet and does not apply
-to the current version.  We plan to update this guide soon.  Until
-then, we are leaving this guide here and hope it will be useful for those
-who wish to learn more about the components that make up OpenVNet.
 
+Welcome to the world of OpenVNet! With this installation guide we help you
+create a very simple yet innovative virtual network environment.
 
-Installation Requirements
--------------------------
+![vnet_minimum](https://www.dropbox.com/s/dezarv561fg7sdj/vnet_minimum.png?dl=1)
 
-### System Requirements
+On a given physical server (here named as server1) there is one virtual network whose network address is 10.100.0.0/24 and 2 virtual machines joining it.
+The orange circles describe OpenVNet's ruby processes; vna, vnmgr and webapi.
+See architecture for more detail of how the OpenVNet works.
+
+
+## Requirements
 
-+ RHEL 6.4
-+ Kernel 2.6.32-358.6.2.el6.x86_64
 
 ### Network Requirements
 
 + Local Area Network (LAN)
 + Internet connection
 
-Pre-setup
----------
+## Installation
+
+### Install OpenVNet Packages
 
-Download the openvnet.repo file and put it to your /etc/yum.repos.d/ directory.
+Download the openvnet.repo file and put it to your `/etc/yum.repos.d/` directory.
 
     # curl -o /etc/yum.repos.d/openvnet.repo -R https://raw.githubusercontent.com/axsh/openvnet/master/openvnet.repo
```

Comment:
This command returned: `curl: (23) Failed writing body (0 != 112)`. In the curl man page I found that this error means curl couldn't write to the file system.

Perhaps it would be a good idea to mention that these commands need to be executed with root privileges.